fcl_product_components/components/gne_reference_pane/gne-reference-pane.tpl.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * Airline name and logo pane template file.
+ *
+ * Theme hook suggestions are available in the format:
+ * airline-logo-pane--CATEGORY-NAME.tpl.php
+ * e.g. airline-logo-pane--air-transporation.tpl.php.
+ *
+ * @var $airline_name string
+ *   The product airline name.
+ *
+ * @var $airline_logo array
+ *   The product airline logo renderable array.
```

Comment:
This file is made for GNE reference so airline logo should be replaced with GNE reference.